src/components/UI/Cards.styled.js

```diff
@@ -1,60 +1,51 @@
 import styled from 'styled-components';
 
-export const CardContainer = styled.li`
+export const CardWrapper = styled.ul`
 	display: flex;
-	align-items: center;
 	flex-direction: column;
+	align-items: center;
+	text-align: start;
+	width: 90%;
+	height: 400px;
+	background-color: black;
+	margin-top: 60px;
+	gap: 20px;
+	margin-left: 20px;
 	border-radius: 20px;
-	height: 50rem;
-	width: 100%;
-	text-align: center;
-	line-height: 30px;
-	color: black;
 	border: solid palevioletred;
-	border-width: 0.3rem;
-	padding: 6rem;
-	background-color: black;
-	margin: 2rem;
-	flex-grow: 5;
 `;
 
-export const WrapperComment = styled.input`
-	display: flex;
-	align-items: flex-start;
-	flex-wrap: wrap;
+export const WrapperComment = styled.textarea`
 	border: solid palevioletred;
-	min-height: 100px;
-	width: 70%;
-	font-size: 1.6rem;
-	word-wrap: wrap;
-	text-align: start;
-	position: absolute;
+	height: 100px;
+	width: 280px;
+	font-family: 'Courier New', Courier, monospace;
+	font-weight: bolder;
 `;
 
-export const CardWrapper = styled.ul`
+export const WrapperButton = styled.div`
 	display: flex;
-	justify-content: column;
-	color: black;
-	margin-top: 6rem;
-	text-align: start;
-	width: 100%;
+	gap: 15px;
+	margin-left: -4px;
```

Comment:
Negative margins are usually not recommended. Is there a different way to achieve your styling?

Reply:
Took it out